### DISCUSSION TITLE
How to custom page preview modal close action color?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [pboivin/filament-peek#135] [Q&A] [answered]
<img width="194" alt="Screenshot 2024-08-25 at 16 30 37" src="https://github.com/user-attachments/assets/778794ba-52a8-4d0f-b61c-4d81cf4086dc">

**Comment by pboivin** [comment, marked as answer]
You can use custom CSS or override this Blade partial: 
https://github.com/pboivin/filament-peek/blob/2.x/resources/views/partials/modal-actions.blade.php
